setup.py

```diff
@@ -135,7 +135,7 @@ def check_version():
 
     'package_data'      :  {'': ['*.sh', '*.json', 'VERSION', 'SDIST']},
 
-    'install_requires'  :  ['radical.pilot', 'pika'],
+    'install_requires'  :  ['radical.pilot', 'pika', 'pandas', 'numpy', 'matplotlib'],
```

Comment:
Are those really needed to run EnTK, or is that a convenience for later data analysis?